src/Entity/Exercise.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ExerciseRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass=ExerciseRepository::class)
+ */
+class Exercise
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $name;
+
+    /**
+     * @ORM\Column(type="integer")
+     * @Assert\LessThanOrEqual(6)
+     * @Assert\GreaterThanOrEqual(1)
+     */
+    private $intervals;
+
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $isPrivate;
```

Comment:
@daria-sieroshtan 
in this app it does not matter but generally, you should always set a default value for booleans.
Nullable boolean = 2 bits, non-nullable-boolean = 1 bit
This makes DB queries significantly faster